sprints/sustainability/models.py

```diff
@@ -85,8 +85,8 @@ def generate_expenses(self, accounts: List[Account]) -> List[SustainabilityAccou
         ]
 
         # Use multiprocessing for parallel API requests for accounts
-        pool = Pool(processes=settings.MULTIPROCESSING_POOL_SIZE)
-        results = [pool.apply_async(SustainabilityAccount, args=arg) for arg in args]
-        output = [p.get() for p in results]
+        with Pool(processes=settings.MULTIPROCESSING_POOL_SIZE) as pool:
+            results = [pool.apply_async(SustainabilityAccount, args=arg) for arg in args]
+            output = [p.get() for p in results]
```

Comment:
This is not strictly related to frontend, but without closing the processes properly we won't be able to run too many queries.

Reply:
Nice fix :+1: